### PR TITLE
Acrntrace fix pr

### PR DESCRIPTION
**Pull request by KaigeFu**
Tracked-On: #4175

### CODE REVIEW

**Comment by KaigeFu** [comment]
@lyan3 Please review this PR. Thanks.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by KaigeFu** [comment]
Tracked-On: #4175

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: misc/tools/acrntrace/acrntrace.c [hunk 1]
```diff
@@ -50,6 +50,7 @@ static void display_usage(void)
 	       "\t-i: period_in_ms: specify polling interval [1-999]\n"
 	       "\t-t: max time to capture trace data (in second)\n"
 	       "\t-c: clear the buffered old data\n"
+	       "\t-r: capture the buffered old data instead of clearing it\n"
```
Comment (lyan3) [inline misc/tools/acrntrace/acrntrace.c]: Pls mark "-c" deprecated. Thx.

Comment (KaigeFu) [reply misc/tools/acrntrace/acrntrace.c]: Will do.

**Comment by KaigeFu** [comment]
@lyan3 PR updated. Please help review.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by wenlingz** [comment]
@dbkinder please help approve this PR doc part, thanks

File: misc/tools/acrntrace/scripts/vmexit_analyze.py [hunk 2]
```diff
@@ -89,44 +89,55 @@ def parse_trace_data(ifile):
...
-                    TSC_BEGIN = tsc
-                    tsc_exit = tsc
-                    TOTAL_NR_EXITS = 0
+            if event != VM_EXIT:
+                continue
 
-                tsc_enter = tsc
-                TSC_END = tsc_enter
-                tsc_last_exit_period = tsc_enter - tsc_exit
+            # We found it and should seek back one line as we will read it in the following loop
```
Comment (deb-intel) [inline misc/tools/acrntrace/scripts/vmexit_analyze.py]: Does "it" refer to vmexit? If so, should we use vmexit?

Comment (KaigeFu) [reply misc/tools/acrntrace/scripts/vmexit_analyze.py]: "it" refers to "first vmexit". I will update the PR to use "the first vmexit".

**Comment by KaigeFu** [comment]
@deb-intel I have updated the PR. Please help review. Thanks.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by KaigeFu** [comment]
@deb-intel @dbkinder @NanlinXie , Could you review and approve this patchset? Thanks in advance.

File: misc/tools/acrntrace/scripts/vmexit_analyze.py (hunk 2 above)
Comment (deb-intel) [reply misc/tools/acrntrace/scripts/vmexit_analyze.py]: @KaigeFu Great, thanks!

**Comment by deb-intel** [comment]
@NanlinXie Looks good. I am ready to merge the PR when you are. Thanks.

**Pull request merged**